modules/admin_manual/pages/release_notes.adoc

```diff
@@ -19,6 +20,108 @@
+* In addition to the option "_Allow username autocompletion in share dialog_" in the global 'Sharing' settings, users can now autonomously decide to opt-out from autocompletion to protect their privacy. When enabled, other users need to enter a user's full identifier in order to be able to share with them.
+In contrast to the *Pending Shares* options, this option is not a general override but an opt-out, meaning it can only be used when "_Allow username autocompletion in share dialog_" is enabled. Users find the new checkbox in the 'Sharing' settings panel of personal settings.
+
+=== Other notable changes
+
+- Added email footer with motto in email for changing password
+If you use customized email templates, it is necessary to adapt those to incorporate the footer. Please compare the original templates with your custom templates (`core/templates/lostpassword/notify.php` and `core/templates/lostpassword/altnotify.php`).https://github.com/owncloud/core/issues/34467
+- Repair steps can now be executed individually in case one would need to be run again
+Repair steps are employed to clean up and resolve issues from former versions. Usually they run during upgrades but some scenarios make it necessary to run them again. To save time when only specific steps need to be taken, administrators can now individually execute them using `occ maintenance:repair --list` and `occ maintenance:repair --single "<repair step>"`.
+- Command for first run wizard to reset for all users
+In some cases administrators customize the First Run Wizard in order to distribute information to users. Using `occ firstrunwizard:reset-all` you can reset the popup for that it will appear for each user upon their next login.
+- Added checkboxes to hide quota and password in user management
+The columns in user management have been made more flexible. Using the bottom left cog wheel you can now hide/show the columns for _Quota_ and _Password_.
+- By default the "apps-external" directory is included in config.php during installation
+For new installations there will be two apps directories to distinguish the bundled apps from the apps that were installed or updated by the administrator. Existing installations will not change but, generally, this separation is recommended in all scenarios as it makes upgrading easier and less error-prone.
+- Added `occ files:scan` `--group` and `--groups` options
+The `occ files:scan` command is used to scan resources on the storage and make them available in ownCloud. While previously it could only be used for all or single users, you can now also execute it for groups of users.
```

Comment:
The `--groups` option was already there (and is still there). You could put a single group, or a comma-separated list of groups. But you could not specify a group name that contained a comma.
Now you can alternatively use the --group option:
```
--group "Admin,Sydney" --group "Finance,Amsterdam" --group "abc123"
```
to specify groups one-by-one, allowing any characters in the group name.